core/src/main/scala/org/apache/spark/sql/store/StoreUtils.scala

```diff
@@ -132,12 +128,10 @@ object StoreUtils extends Logging {
       table: String,
       schema: Option[StructType],
       partitions: Int,
-      connProperties: ConnectionProperties): Map[InternalDistributedMember,
-      BlockManagerId] = {
+      connProperties: ConnectionProperties): Unit = {
     // TODO for SnappyCluster manager optimize this . Rather than calling this
-    val blockMap = new StoreInitRDD(sqlContext, table,
-      schema, partitions, connProperties).collect()
-    blockMap.toMap
+    new StoreInitRDD(sqlContext, table, schema, partitions, connProperties)
```

Comment:
This PR addresses the second part of that goal i.e. updating the block map outside StoreInitRDD. Will file another sub-task/PR for the first part i.e. getting rid of the need to boot up the store in embedded mode via StoreInitRDD. (I remember some tests failing when StoreInitRDD was removed, even with these changes).